### DISCUSSION TITLE
Possibility to open source the front of https://copilot.github.com/ ?

### DISCUSSION THREAD

**Discussion by FournyP** [community/community#15513] [Copilot Conversations]
Hi everyone,

I want to create a portfolio website and I want to fork the front of the copilot website because their is a lot of feature I want for my portfolio.

So my question is, can the copilot website be open source or just the components ? 

Have a good day

**Comment by PGSCOM** [comment]
It’s a good idea. It can also be useful to me.

**Comment by FournyP** [comment]
Up

**Comment by FournyP** [comment]
Hi everyone

Now, https://copilot.github.com has been closed because it redirect directly on the github.com website

So it's possible to make it open source ?

Have a good day
